src/ngx_http_lua_module.c

```diff
@@ -23,7 +23,7 @@
 #include "ngx_http_lua_initby.h"
 #include "ngx_http_lua_initworkerby.h"
 #include "ngx_http_lua_probe.h"
-
+#include "ngx_http_lua_semaphore.h"
 
```

Comment:
Style: 2 blank lines needed here. Yes, we use blank lines and spaces very carefully in the NGINX coding style.